### ISSUE TITLE
Lighting "blend" over to the next column

### ISSUE THREAD

**Issue by Codetector1374** [OpenAnnePro/AnnePro2-Shine#17]
# Problem Description
When a LED that is turned off is to the right (including wrapping around) of a brightly lit key, the lighting will "spill" over to the key that has lighting turned off.

# How to reproduce
Have only one light in a row turned on or off. You might need to remove keycap to see the effect as the "bleed" over key is very dim. (sometimes it blinks which is very annoying)

**Comment by Codetector1374** [comment]
I tried to resolve this problem but yet to find a good solution. I managed to improve it a little bit by changing the order things happen on a columnCallback cycle. Here's the new order:

1.  Disable the old column
2.  Setup the row GPIO for the next column
3. Turn on the next column power

This seems to make the "bleed" over dark enough (esp with a keycap) that it is easily distinguishable from a fully lit key.
